tests/storage-buckets

```diff
@@ -11,6 +11,20 @@ if ! hasNeededAPIExtension storage_buckets; then
   exit 0
 fi
 
+
+# Grab the architecture so we can install the correct pre-built minio.
+arch="$(uname -m)"
+if [ "${arch}" = "aarch64" ]; then
+  arch="arm64"
+elif [ "${arch}" = "ppc64el" ]; then
+  arch="ppc64le"
+fi
+
+if ! echo "amd64 arm64 ppc64le s390x" | grep -q "${arch}" ; then
```

Comment:
It seems like `armhf` is also supported https://dl.min.io/server/minio/release/linux-arm/
I think you'll also need some `uname -m` mangling here too though.

In the end, all our arches except for `riscv64` are supported so maybe you'd prefer to invert the checking logic.

Reply:
Even when we vendored `minio`, `riscv64` was not supported.

Reply:
I'm not sure where to find "our architctures" so I just tried grepping the codebase and I see some references to `ppc64` and `mips64le` which aren't supported by minio (though they do support `ppc64le` and `mips64`). Do we run tests for these?

Reply:
https://launchpad.net/~canonical-lxd/+snap/lxd-latest-edge/+edit show you the list of arches for which we build the snap for at the bottom

Reply:
Thanks!